OSSM-6063 [DOC] OpenShift Service Mesh Console (OSSMC) plugin Integration (#88446)
openshift/openshift-docs: modules/ossm-uninstall-console-plugin-ocp-cli.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/observability/kiali/ossm-console-plugin-assembly.adoc

:_mod-docs-content-type: PROCEDURE
[id="ossm-uninstall-console-plugin-ocp-cli_{context}"]

> = Uninstalling {SMPluginShort} via {ocp-short-name} CLI

Reviewer: ```suggestion
= Uninstalling {SMPluginShort} by using the CLI
```

> You can uninstall the {SMPlugin} using the {ocp-short-name} CLI.

Reviewer: ```suggestion
You can uninstall the {SMPlugin} by using the {oc-first}.
```

.Procedure

> . Remove the OSSMC custom resource (CR) by running the following command:

Reviewer: ```suggestion
* Remove the OSSMC custom resource (CR) by running the following command:
```

+
[source,terminal]
----
$ oc delete ossmconsoles <custom_resource_name> -n <custom_resource_namespace>
----

> . Verify all the CRs are deleted from all namespaces by running the following command:

Reviewer: ```suggestion
.Verification

* Verify all the CRs are deleted from all namespaces by running the following command:
```

// Module included in the following assemblies:
//
// * service-mesh-docs-main/observability/kiali/ossm-console-plugin.adoc

:_mod-docs-content-type: PROCEDURE
[id="ossm-uninstall-console-plugin-ocp-cli_{context}"]
= Uninstalling {SMPluginShort} by using the CLI

> You can uninstall the {SMPlugin} by using the {ocp-short-name} CLI.

Reviewer: ```suggestion
You can uninstall the {SMPlugin} by using the {oc-first}.
```
